themes/digital.gov/layouts/resources/single.html

```diff
@@ -18,27 +18,22 @@ <h1>{{- .Title | markdownify -}}</h1>
       </header>
 
       <div class="usa-in-page-nav-container">
-        <div>
-          {{/* Content */}}
-          <div class="content usa-prose">
-            {{- .Content -}}
-          </div>
-
-          {{/* Touchpoints Form */}}
-          {{- partial "core/touchpoints-form.html" . -}}
+        <div class="content usa-prose">
+          {{- .Content -}}
         </div>
+
         <aside
           class="dg-sidebar usa-in-page-nav"
           data-title-heading-level="h4"
           data-root-margin="-350px 0px -350px 0px"
           aria-label="Related content"
         >
           <div class="dg-related-items" data-related-items>
-            {{/* Related Communities and Services */}}
             {{- partial "core/get_related.html" . -}}
           </div>
         </aside>
       </div>
```

Comment:
### Improvement

It would be ideal to have the component take a setting for displaying the `get_related.html` partial.


```html
      <div class="usa-in-page-nav-container">
        <aside
          class="dg-sidebar usa-in-page-nav"
          data-title-heading-level="h4"
          data-root-margin="-350px 0px -350px 0px"
          aria-label="Related content"
        >
          {{ if $has_sidebar }}
          <div class="dg-related-items" data-related-items>
            {{- partial "core/get_related.html" . -}}
          </div>
          {{ end }}
        </aside>
        <div class="content usa-prose">
          {{- .Content -}}
        </div>
      </div>
```

Reply:
This would entail more refactoring. We can skip this.